plugins/inertia.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Inertia Plugin
+description: Use this to integrate your FormKit forms with Inertia calls.
+---
+
+# Inertia Plugin
+
+:PageToc
+
+The `@formkit/inertia` plugin aims to seamlessly integrate Inertia.js with FormKit forms, leveraging a robust event system that harnesses Inertia.js event callbacks and FormKit plugins for a smooth and powerful web development experience, and here is why:
+
+- Your time should be used on creating forms and backends, not integrations.
+- Having an easy to install and use package, makes it so you don't need to care about packaging and publishing.
+
+::Callout
+---
+type: "warning"
+label: "Inertia Plugin"
+---
+The <code>@formkit/inertia</code> is intended to run on a FormKit form. Your usage of the composable and plugin will be on a per-form basis and each form that requires Inertia integration.
+::
```

Comment:
I think this warning comes a bit early, people won’t understand how it relates to their project until after reading your docs, so lets remove it from here.
```suggestion
```